.kokoro

```diff
@@ -92,8 +92,14 @@ run_bazel() {
     TARGET="//components/..."
   fi
 
+  if [ -n "$XCODE_VERSION" ]; then
+    xcode_versions="--xcode-version $XCODE_VERSION"
+  else
+    xcode_versions="--min-xcode-version $XCODE_MINIMUM_VERSION"
+  fi
+
   ./.kokoro-ios-runner/bazel.sh $COMMAND $TARGET \
-      --min-xcode-version $XCODE_MINIMUM_VERSION \
+      $xcode_versions \
```

Comment:
Is there already support for the new flag (`--xcode-version`) in bazel.sh?

Reply:
Yep!